iOS/Tests/BaseSDKTest.swift

```diff
@@ -104,6 +83,13 @@ class LoginTests: XCTestCase {
ManagerTestCase_Contact_21600929  S SyncManagerTestCase_Contact_39435484  S SyncManagerTestCase_Contact_39883565  S SyncManagerTestCase_Contact_42527071  S SyncManagerTestCase_Contact_43536868  S SyncManagerTestCase_Contact_44546449  S SyncManagerTestCase_Contact_51592337  S SyncManagerTestCase_Contact_54696463  S SyncManagerTestCase_Contact_55085218  S SyncManagerTestCase_Contact_56513612  S SyncManagerTestCase_Contact_61167456  S SyncManagerTestCase_Contact_71855613  S SyncManagerTestCase_Contact_73858564  S SyncManagerTestCase_Contact_76118170  S SyncManagerTestCase_Contact_98648664  TT TestTargetiOS12 TestXCode10  IT Ios12 Testnewos Test stuff  XX Xcode10test Xcode10works  AY Andy Young SVP, Operations  wj web-test justatest  Contacts    Dismiss All Warning: componentWillReceiveProps is deprecated and will be removed in the next major version. Use static getDerivedStateFromProps instead.\\n\\nPlease update the following components: SafeView, Transitioner\\n\\nLearn more about this warning here:\\nhttps:\/\/fb.me\/react-async-component-lifecycle-hooks\")",".otherElements.matching(identifier: \"Search a contact...  TB Tim Barr SVP, Administration and Finance  JB John Bond VP, Facilities  LB Lauren Boyle SVP, Technology  LD Lizzz D'Cruz VP, Production  JD Josh Davis Director, Warehouse Mgmt  SF Sean Forbes CFO  EF Edna Frank VP, Technology  TG Testandroid Gchkvfd  RG Rose Gonzalez SVP, Procurement  AG Avi Green CFO  JG Jane Grey Dean of Administration  TH Testios Hshaha  AJ Ashley James VP, Finance  FL First928848 Last928848  BL Babara Levy SVP, Operations  SP Stella Pavlova SVP, Production  TR Tom Ripley Regional General Manager  JR Jack Rogers VP, Facilities  AS Arthur Song CEO  PS Pat Stumuller SVP, Administration and Finance  S SyncManagerTestCase_Contact_05984046  S SyncManagerTestCase_Contact_06777687  S SyncManagerTestCase_Contact_20980716  S SyncManagerTestCase_Contact_21600929  S SyncManagerTestCase_Contact_39435484  S SyncManagerTestCase_Contact_39883565  S SyncManagerTestCase_Contact_42527071  S SyncManagerTestCase_Contact_43536868  S SyncManagerTestCase_Contact_44546449  S SyncManagerTestCase_Contact_51592337  S SyncManagerTestCase_Contact_54696463  S SyncManagerTestCase_Contact_55085218  S SyncManagerTestCase_Contact_56513612  S SyncManagerTestCase_Contact_61167456  S SyncManagerTestCase_Contact_71855613  S SyncManagerTestCase_Contact_73858564  S SyncManagerTestCase_Contact_76118170  S SyncManagerTestCase_Contact_98648664  TT TestTargetiOS12 TestXCode10  IT Ios12 Testnewos Test stuff  XX Xcode10test Xcode10works  AY Andy Young SVP, Operations  wj web-test justatest  Contacts   \")",".otherElements.matching(identifier: \"Contacts   \")",".otherElements[\"Contacts\"].staticTexts[\"Contacts\"]",".staticTexts[\"Contacts\"]"],[[[-1,4],[-1,3],[-1,2,3],[-1,1,2],[-1,0,1]],[[-1,4],[-1,3],[-1,2,3],[-1,1,2]],[[-1,4],[-1,3],[-1,2,3]],[[-1,4],[-1,3]]],[0]]@END_MENU_TOKEN@*/
```

Comment:
What are those strings? Wouldn't that change if other tests run or contacts are added / deleted?

Reply:
You are correct.  I will check if this can be improved, but this is a limitation of XCUITest with React Native.